<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="PC380387" ignored="3" name="Test" tests="1" failures="1" timestamp="17 May 2018 13:12:33 GMT" time="68.79" errors="0">
  <testcase name="Test_Case_1" time="62.408" classname="Application.Suite.AutomationTestSuite">
    <failure type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at Framework.Handler.Runner.execute(Runner.java:57)
	at Framework.Handler.TestBaseWebAutomation.startExecution(TestBaseWebAutomation.java:51)
	at Application.Suite.AutomationTestSuite.Test_Case_1(AutomationTestSuite.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: Framework.Utilities.FrameworkException: No data found in Input TestData
	at Framework.Utilities.ReusableLibrary.getValue(ReusableLibrary.java:96)
	at Framework.Utilities.ReusableLibrary.enterTxt(ReusableLibrary.java:126)
	at AutomationTestSuite.keywords.Test_Case_1_Keywords.run(Test_Case_1_Keywords.java:32)
	... 20 more
]]>
    </failure>
  </testcase> <!-- Test_Case_1 -->
  <testcase name="Test_Case_2" time="0.0" classname="Application.Suite.AutomationTestSuite">
    <ignored/>
  </testcase> <!-- Test_Case_2 -->
  <testcase name="Test_Case_4" time="0.0" classname="Application.Suite.AutomationTestSuite">
    <ignored/>
  </testcase> <!-- Test_Case_4 -->
  <testcase name="Test_Case_3" time="0.0" classname="Application.Suite.AutomationTestSuite">
    <ignored/>
  </testcase> <!-- Test_Case_3 -->
</testsuite> <!-- Test -->
